text/0000-slice-string-symmetry.md

```diff
@@ -0,0 +1,68 @@
+- Feature Name: `slice_string_symmetry`
+- Start Date: 2015-06-06
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Add some methods that already exist on slices to strings and vice versa.
+Specifically, the following methods should be added:
+
+- `str::chunks`
+- `str::windows`
+- `str::into_string`
+- `String::into_boxed_slice`
+- `<[T]>::subslice_offset`
+
+# Motivation
+
+Conceptually, strings and slices are similar types. Many methods are already
+shared between the two types due to their similarity. However, not all methods
+are shared between the types, even though many could be. This is a little
+unexpected and inconsistent. Because of that, this RFC proposes to remedy this
+by adding a few methods to both strings and slices to even out these two types’
+available methods.
+
+# Detailed design
+
+Add the following methods to `str`, presumably as inherent methods:
+
+- `chunks(&self, n: usize) -> Chunks`: Returns an iterator that yields the
+  *characters* (not bytes) of the string in groups of `n` at a time. Iterator
```

Comment:
“Character” is kinda ambiguous. (Unicode has [four different definitions](http://www.unicode.org/glossary/#character).) Maybe use *`char`s* instead?